packages/@aws-cdk/toolkit-lib/lib/api/hotswap/hotswap-deployments.ts

```diff
@@ -93,6 +94,7 @@ export async function tryHotswapDeployment(
   stackArtifact: cxapi.CloudFormationStackArtifact,
   hotswapMode: HotswapMode,
   hotswapPropertyOverrides: HotswapPropertyOverrides,
+  hotswapOperationOptions?: HotswapOperationOptions,
```

Comment:
It might be easier to start combining these two (`hotswapPropertyOverrides` and `hotswapOperationOptions`) into a single object.